### PR TITLE
#1551 SfBottomNavigation fixes

### LINKED ISSUE #1551: [BUG] SfBottomNavigation - adjust the same wrapper size for the items
**Describe the bug**
Currently SfBottomNavigationItem size depends on inner icon size but all of them should have the same size.
The navigation looks like this:
image

 and should be like this:
image

**How to reproduce**
See the component on Storybook and check in devtools.

**Expected behavior**
Every SfBottomNavigationItem component size should be the same.

**Actual behavior**

**Code examples**

**Screenshots**

**Additional context**
Anything you would like to add but was not sure which section it fits. For example: tech stack used (Nuxt, Vue, Gridsome, React, etc)

### PR DESCRIPTION
**Pull request by mayashavin**
# Related issue
Closes #1551

# Scope of work
- Fix problem with width not expanded to the container for SfBottomNavigationItem, which causes clicking problem.
- Remove all the component-specific CSS variables.

# Checklist

- [x] No commented blocks of code left
- [x] Run tests and docs
- [x] Self code-reviewed
- [x] Changes documented 

If applicable:

- [x] I followed [composition rules](https://docs.storefrontui.io/contributing/coding-guidelines.html#component-rules) for my component
- [x] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

File: packages/shared/styles/components/organisms/SfBottomNavigation.scss [hunk 1]
```diff
@@ -4,55 +4,43 @@
...
-  height: var(--bottom-navigation-height, 3.75rem);
-  padding: var(--bottom-navigation-padding, 0 var(--spacer-sm));
-  background: var(--bottom-navigation-background, var(--c-white));
-  box-shadow: var(
-    --bottom-navigation-box-shadow,
-    0px -2px 10px rgba(var(--_c-gray-secondary-base), 0.15)
-  );
+  height: 3.75rem;
+  background: var(--c-white);
+  box-shadow: 0px -2px 10px rgba(var(--_c-gray-secondary-base), 0.15);
```
Comment (leomp12) [inline packages/shared/styles/components/organisms/SfBottomNavigation.scss:14]: shouldn't we have global css vars for box shadow and apply it here?

Comment (leomp12) [inline packages/shared/styles/components/organisms/SfBottomNavigation.scss:14]:
```suggestion
  box-shadow: var(--box-shadow);
```

File: packages/shared/styles/components/organisms/SfBottomNavigation.scss [hunk 2]
```diff
@@ -4,55 +4,43 @@
...
-    var(--font-weight--light),
-    var(--font-size--xs),
-    1,
-    var(--font-family--primary)
-  );
+  padding: var(--spacer-xs) 0 var(--spacer-xs) 0;
+  color: var(--c-link);
+  font-weight: var(--font-weight--light);
+  font-size: var(--font-size--xs);
+  font-family: var(--font-family--primary);
```
Comment (leomp12) [inline packages/shared/styles/components/organisms/SfBottomNavigation.scss:29]: not directly related to this pr, but is there a reason to use `--spacer-xs` and `--font-size--xs`? why not `--font-size-xs`, `--font-family-primary`, `--font-weight-light` (...)? i don't think we should follow bem conventions on css vars :thinking:

File: packages/shared/styles/components/organisms/SfBottomNavigation.scss (hunk 1 above)
Comment (leomp12) [reply packages/shared/styles/components/organisms/SfBottomNavigation.scss:14]: something like that...

File: packages/shared/styles/components/organisms/SfBottomNavigation.scss [hunk 3]
```diff
@@ -4,55 +4,43 @@
...
-    var(--spacer-xs) 0 var(--spacer-xs) 0
-  );
-  color: var(--bottom-navigation-item-color, var(--c-link));
-  @include font(
-    --bottom-navigation-item-font,
-    var(--font-weight--light),
-    var(--font-size--xs),
-    1,
-    var(--font-family--primary)
-  );
```
Comment (leomp12) [inline packages/shared/styles/components/organisms/SfBottomNavigation.scss:42]: so much better without all those vars with big names :smile:

File: packages/shared/styles/components/organisms/SfBottomNavigation.scss (hunk 2 above)
Comment (mayashavin) [reply packages/shared/styles/components/organisms/SfBottomNavigation.scss:29]: @leomp12 TBH no, I would prefer not to use --font-size--xs and --spacer-xs. But that should be in a separate PR where we rename all the global CSS variables. These are the current global defined one :).

File: packages/shared/styles/components/organisms/SfBottomNavigation.scss (hunk 1 above)
Comment (mayashavin) [reply packages/shared/styles/components/organisms/SfBottomNavigation.scss:14]: We should. Can you open a PR for global css variables? :)

Comment (leomp12) [reply packages/shared/styles/components/organisms/SfBottomNavigation.scss:14]: @mayashavin yeah, i'll give a try

**Review by justyna-13 — APPROVED** [review]

**Review by Baroshem — APPROVED** [review]
Looks good to me!

**Comment by Baroshem** [comment]
@mayashavin 

Please, rebase your branch to be up to date with `develop` and you are good to go ;)

**Pull request merged**